jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/resources/js/lib/backbone-amd.js

```diff
@@ -0,0 +1,5 @@
+define('slack/backbone', [
+    'jira/backbone-1.3.3'
+], function (Backbone) {
+    return Backbone;
+});
```

Comment:
Github complains about missing endline here. I don't know why it's needed, but maybe add it if Github wants it.

Reply:
added. thanks